### DISCUSSION TITLE
Correct the vcpkg readme regarding additional configuration

### DISCUSSION THREAD

**Discussion by PJBoy** [microsoft/vcpkg#20283] [Ideas]
Seems that the issues raised in #18822 and #18760 are here to stay, which is fine, we can just switch to CMake or the ridiculous copy+paste based workaround from the former issue's thread. However, we should remove claim from the readme that no additional configuration is required for non-CMake projects, as it is demonstrably not true and seemingly impossible to fix.

The vcpkg readme promises: 
> After this, you can now create a New non-CMake Project (or open an existing one). All installed libraries are immediately ready to be #include'd and used in your project without additional configuration.

_Originally posted by @PJBoy in https://github.com/microsoft/vcpkg/issues/18760#issuecomment-890053499_

**Comment by MehradN** [comment]
They've closed the issue as completed without removing the false statement from their readme or any explanation about why they won't remove it. It seems like they are going to continue with their false advertising.

**Reply by PJBoy** [reply]
Yeah it's pretty miserable. I'm guessing the reason they don't remove the claim is because not needing to use CMake is the biggest draw vs Conan for many(?)

I've had to inform a few people about this on the internet who've run into the same kind of problem as me. Not a pleasant surprise to find out you need to learn CMake _anyway_ (except you'll have to do it _after_ your project has started).

Perhaps they're just trying to maintain the status quo that building C++ code is absurdly hard 😂
